src/SIL.LCModel.Core/Phonology/PhonEnvRecognizer.cs

```diff
@@ -16,11 +18,11 @@ namespace SIL.LCModel.Core.Phonology
 	/// </summary>
 	public class PhonEnvRecognizer
 	{
-		private bool m_fSuccess;
-		private string m_sErrorMessage;
-		private PhonEnvParser m_parser;
-		private string[] m_saNaturalClasses;
-		private string[] m_saSegments;
+		private bool m_fSuccess = false;
+		private string m_sErrorMessage = "";
+		private PhonEnvParser? m_parser;
```

Comment:
I say we just new up the parser here. It seems like it will be created when it's needed, but there's no arguments required so we should just make it here so it's never null.